### PR TITLE
Fixes from SpikeInterface CI

### PR DESCRIPTION
**Pull request by alejoe91**
A couple of small fixes that came up when addin tests in SI:

- BiocamRawIO: remove squeeze when returning traces
- AxonaRawIO: construct list of channels also when `channel_indexes` is a `slice`

### CODE REVIEW

**Comment by alejoe91** [comment]
@samuelgarcia @JuliaSprenger

File: neo/rawio/axonarawio.py [hunk 1]
```diff
@@ -304,6 +304,9 @@ def _get_analogsignal_chunk(self, block_index, seg_index, i_start, i_stop,
             i_stop = bin_dict['num_total_samples']
         if channel_indexes is None:
             channel_indexes = [i for i in range(bin_dict['num_channels'])]
+        if isinstance(channel_indexes, slice):
```
Comment (samuelgarcia) [inline neo/rawio/axonarawio.py:307]: I would put elif

Comment (alejoe91) [inline neo/rawio/axonarawio.py:307]:
```suggestion
        elif isinstance(channel_indexes, slice):
```

**Comment by samuelgarcia** [comment]
Thanks for the fix.
This reinforce friendship between peoples.

**Pull request merged**